Woof/Woof/View/SitterFlow/SitterWalkingsView/SitterWalkingCardView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+/// A view representing a card displaying details of a walking.
+struct SitterWalkingCardView: View {
+    /// The walking information to display
+    let walking: Walking
```

Comment:
Maybe it makes sense to implement a view model for the view?

Reply:
Yes, the view looks complex, it's a good idea to move data formatting into the view model.